eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -342,7 +335,22 @@ stages:
               arguments: >
                 -ReleasePlanWorkItemId ${{ parameters.ReleasePlanWorkItemId }}
                 -PullRequestUrl "https://github.com/Azure/$(SdkRepoName)/pull/$(Submitted.PullRequest.Number)"
-                -Status "$(SdkPrStatus)"
+                -Status "draft"
+                -SdkRepoName "$(SdkRepoName)"
+
+          - task: AzureCLI@2
+            displayName: Update SDK generation status as failed
```

Comment:
The step name looks confusing to me. Should that be release plan related either?